### DISCUSSION TITLE
Quake II

### DISCUSSION THREAD

**Discussion by chrisstonehill** [eduardofilo/RG350_adam_image#267] [Ports]
A while back I asked about getting Quake II running on the last 1.3 version of Adam image (which is what I am still running on my 350m).

Did I see somewhere, maybe even for the 1.4 versions, thus I guess maintained for the current 2.0 version?...that this was much more integrated and straightforward now? (maybe it wasn't even possible with 1.3 versions?) I have the files I need, I think...having bought Quake II from GOG...though on my laptop there doesn't seem to be any music. There is supposed to be some isn't there? I can't remember from when I bought the game in a UK news agent (kind of, WHSmiths) in the town where I went to school and installed it on my family's PC...just as my interest in gaming waned (it is now back with a vengeance, since buying a SNES mini in 2018). It did run, as a port on another complete image that I tried before the Adam image, CBMT 5, but again without music.

The music files are there. Just read something about copying them in to the baseq2 folder, but this made no difference. Listened to the music. REALLY would like it to play in the game! It's great. Did Trent Reznor have something to do with it?...ah no, that was for the first game.

Not yet found the time for jumping from 1.3 to 2.0 ...pretty much happy with how everything else is working in 1.3.something, but I expect there are some other benefits, other than maybe getting Quake 2 working?

Also, just got a Retroid Pocket 2+ up and running. A lot of fun and great being able to emulate another generation of consoles, mainly very well...but also makes me appreciate even more the absolutely stunning job that has been done by all behind the Adam image. Will continue jumping between the two.

**Comment by a-dekker** [comment]
For RA Quake 2 xatrix I have trackXX.ogg files in a music subfolder. Works ok here.

**Reply by chrisstonehill** [reply]
OK...I will tinker a bit...and working fine in the latest Adam image? Maybe a Quake folder is made on the external SD card now?...to just drop the relevant files in to?

**Reply by a-dekker** [reply]
I have the music folder as subfolder in the location where pak0.pak resides. And I do use the same Retroarch version as Adam 2.0 does.

**Reply by chrisstonehill** [reply]
Yeah...I tried copying the music folder in to the baseq2 folder, where the pak files are (on my laptop). Not quite sure what it runs in with the files downloaded from GOG. Doom from them opens up in DOSbox, Quake 2, I'm not sure (just tried opening it, seems like a custom GOG.com Quake II launcher). Hopefully when I update to the latest Adam image and try Quake 2 again on my 350m I will find that the music does function...remembering that I may well need to relocate the music folder. THAT will be motivation to do the update (otherwise I am very satisfied with how everything else I want to play functions in 1.3.something Adam image, making me loathe to change anything, when everything seems to work)...but hopefully an update to 2.0 doesn't need to be too much of a hassle...just need to remind myself how save states get carried over. I have a few quite vital ones, deep in to some games :-D

**Comment by eduardofilo** [comment]
Quake II support was added in Adam 1.4 so you need that version at least. Therefore, it would be better to go directly to 2.0.
Savestates from v2.0 onwards will be kept on the external SD so it will be easier to update. But between 1.x and 2.0 you have to migrate the savestates. For this purpose a small utility was made and is included in the 2.0 releases along with this video on how to use it.
https://www.youtube.com/watch?v=lMnhQhVmzKs

**Reply by chrisstonehill** [reply]
Thanks Eduardo! At some point during the summer I will make the jump and have portable Quake 2 with the 350m's lovely twin sticks. Just been applying some tweaks to an rp2+ I got recently. Interesting experimenting with the Android approach too. The right hand "slider" instead of thumbstick, that everyone hates, is pretty horrible. I haven't really needed it, but was trying making use of it with TATE, vertical orientated arcade games. Not very nice! Looks a bit odd but I actually I might find using the d-pad up above the (vertical) screen and the face buttons below preferable...also means the fingers of my two hands don't get so tangled up with each other, as they do when using the controls just below the vertical screen
